kt/plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/projectExt/setupTasks.kt

```diff
@@ -0,0 +1,85 @@
+package godot.gradle.projectExt
+
+import godot.gradle.tasks.android.checkAndroidJarAccessibleTask
+import godot.gradle.tasks.android.checkD8ToolAccessibleTask
+import godot.gradle.tasks.android.createMainDexFileTask
+import godot.gradle.tasks.android.packageBootstrapDexJarTask
+import godot.gradle.tasks.android.packageMainDexJarTask
+import godot.gradle.tasks.createBuildLockTask
+import godot.gradle.tasks.deleteBuildLockTask
+import godot.gradle.tasks.generateEntryServiceFileTask
+import godot.gradle.tasks.graal.checkNativeImageToolAccessibleTask
+import godot.gradle.tasks.graal.checkPresenceOfDefaultGraalJniConfigTask
+import godot.gradle.tasks.graal.createGraalNativeImageTask
+import godot.gradle.tasks.packageBootstrapJarTask
+import godot.gradle.tasks.packageMainJarTask
+import godot.gradle.tasks.setupBuildTask
+import godot.gradle.tasks.setupCleanTask
+import org.gradle.api.Project
+
+fun Project.setupTasks() {
+    afterEvaluate {
+        with(it) {
+            val createBuildLockTask = createBuildLockTask()
+            val deleteBuildLockTask = deleteBuildLockTask()
+            val generateEntryServiceFileTask = generateEntryServiceFileTask()
+            val packageBootstrapJarTask = packageBootstrapJarTask(
+                createBuildLockTask = createBuildLockTask,
+                deleteBuildLockTask = deleteBuildLockTask
+            )
+            val packageMainJarTask = packageMainJarTask(
+                createBuildLockTask = createBuildLockTask,
+                deleteBuildLockTask = deleteBuildLockTask,
+                generateEntryServiceFileTask = generateEntryServiceFileTask
+            )
+
+            // START: android specific tasks
+            val checkD8ToolAccessibleTask = checkD8ToolAccessibleTask()
```

Comment:
I was expecting an `if (isAndroidExportConfigured())` statement here, what happens if the user doesnt' want to build for android?

Reply:
This is currently taking care of in the setup of the `build` task:
```kotlin
if (godotJvmExtension.isAndroidExportEnabled.get()) {
    finalizedBy(packageBootstrapDexJarTask, packageMainDexJarTask)
}
```

Reply:
Ahh, weird way of doing it - we don't really need to configure the android/native image tasks when they are disabled?

Reply:
Anyway, not a big issue. This will change once I'm done with the refactor.

Reply:
:+1: